src/interfaces/routes.ts

```diff
@@ -485,6 +485,29 @@ export interface AdjustLeverageResponse {
   maxNotionalValue: string;
 }
 
+export interface SubAccountRequest {
+  accountAddress: string;
+  accountsToRemove: Array<string>;
+}
+
+export interface SignedSubAccountRequest extends SubAccountRequest {
+  signedTransaction: string;
+}
+
+export interface SubAccountResponse {
+  userAddress: string;
+  txIndex?: number;
+  logIndex?: number;
+  blockNumber?: number;
+  id: number;
+  symbol: string;
+  subAccountAddress: string;
+  approved?: boolean;
```

Comment:
Do we need a separate flag for this? I think this should be confirmed by the http status code of the response